test/test_raw_api.py

```diff
@@ -0,0 +1,94 @@
+######################################################################
+#
+# File: b2/test_raw_api.py
+#
+# Copyright 2018 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+from __future__ import print_function
+
+import six
+
+from b2.raw_api import B2RawApi
+from b2.b2http import B2Http
+from b2.exception import UnusableFileName
+from .test_base import TestBase
+
+if six.PY3:
```

Comment:
I don't think you need this hack.  `six.unichr()` is the function you want.

My PyCharm warns about redefining `unichr`.